Bullwinkle.class.nut

```diff
@@ -147,7 +151,6 @@ class Bullwinkle {
             "name": command,
             "data": data,
             "ts": ts,
-            "tries": 0,
```

Comment:
There is a comment in the other thread that ```message.retry and message.tries are items that I don't expect anyone would be using,```. These are things I use frequently (in almost every project) to allow for [3] retries before failing with an error message. Be careful.